storage/StorageManager.cpp

```diff
@@ -487,23 +481,18 @@ MutableBlockReference StorageManager::getBlockInternal(
     const block_id block,
```

Comment:
Given the complexity of this method, could we have a comment for this whole function to explain the flow?